### PR TITLE
chore: 增加 rejected label action

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [x] 增加 rejectec label action 

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by xingwanying — APPROVED** [review]
LG

File: .github/workflows/issue-labeled.yml [hunk 1]
```diff
@@ -89,6 +89,18 @@ jobs:
...
+        if: github.event.label.name == '🙈 rejected'
+        uses: actions-cool/issues-helper@main
+        with:
+          actions: 'create-comment,close-issue'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            你好 @${{ github.event.issue.user.login }}，经过我们的反复讨论, 你的需求不适合直接在添加到S2中, 你可试着自行通过 [hooks](https://s2.antv.vision/zh/docs/manual/advanced/custom/hook) 定制，感谢你的理解。
+
+            Hello, @${{ github.event.issue.user.login }}, your feature request isn't suitable for implementing directly in s2 after our repeated discussion, you could try to customize it through [hooks](https://s2.antv.vision/zh/docs/manual/advanced/custom/hook). Thanks so much for your understanding.
```
Comment (xingwanying) [inline .github/workflows/issue-labeled.yml:106]: after careful consideration?

File: .github/workflows/issue-labeled.yml [hunk 2]
```diff
@@ -89,6 +89,18 @@ jobs:
...
+        
+      - name: Rejected
+        if: github.event.label.name == '🙈 rejected'
+        uses: actions-cool/issues-helper@main
+        with:
+          actions: 'create-comment,close-issue'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            你好 @${{ github.event.issue.user.login }}，经过我们的反复讨论, 你的需求不适合直接在添加到S2中, 你可试着自行通过 [hooks](https://s2.antv.vision/zh/docs/manual/advanced/custom/hook) 定制，感谢你的理解。
```
Comment (lijinke666) [inline .github/workflows/issue-labeled.yml:104]: 直接在 => 直接

Comment (lijinke666) [reply .github/workflows/issue-labeled.yml:104]: 需求过于定制化, 不适合添加在 S2 中, 请根据开发的API自行实现你自己的业务

File: .github/workflows/issue-labeled.yml [hunk 3]
```diff
@@ -89,6 +89,18 @@ jobs:
...
           issue-number: ${{ github.event.issue.number }}
           body: |
-            你好 @${{ github.event.issue.user.login }}，经过我们的反复讨论, 你的需求现已被采纳, 我们会排期开发, 请关注后续发布日志.
+            你好 @${{ github.event.issue.user.login }}，经过我们的反复讨论, 你的需求现已被采纳, 我们会排期开发, 请关注后续发布日志。
 
-            Hello, @${{ github.event.issue.user.login }}, your feature request have been accepted After our repeated discussion. We will schedule the development, please pay attention to the follow-up release logs
+            Hello, @${{ github.event.issue.user.login }}, your feature request have been accepted after our repeated discussion. We will schedule the development, please pay attention to the follow-up release logs.
+        
+      - name: Rejected
+        if: github.event.label.name == '🙈 rejected'
```
Comment (lijinke666) [inline .github/workflows/issue-labeled.yml:97]: 🙅🏻‍♀️ rejected

**Review by lijinke666 — APPROVED** [review]

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**